### PR TITLE
docs(pagination): Adds React documentation for Pagination component

### PR DESCRIPTION
**Pull request by diegonvs**

### CODE REVIEW

File: next.clayui.com/content/docs/components/pagination.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- [Variants and Best Pratices](#variants-and-best-pratices)
+- [API](#api)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+Check the <a href="https://liferay.design/lexicon/">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/pagination/">Pagination Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+## Best Pratices
```
Comment (bryceosterhaus) [inline next.clayui.com/content/docs/components/pagination.mdx]: I'm not sure how useful this section is, just seems like a re-iteration of the API. What do you think @matuzalemsteles?

Comment (matuzalemsteles) [reply next.clayui.com/content/docs/components/pagination.mdx]: Yes, I think we can limit the information here a bit and we can start doing what we talked about in Slack.

**Comment by matuzalemsteles** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by matuzalemsteles** [comment]
Resent #2090

**Pull request closed without merging**